web-components/src/merch-card.js

```diff
@@ -110,10 +111,12 @@ export class MerchCard extends LitElement {
 
     updated(changedProperties) {
         if (
-            changedProperties.has('badgeBackgroundColor') &&
+            (changedProperties.has('badgeBackgroundColor') || changedProperties.has('borderColor')) &&
             this.variant !== 'twp'
         ) {
-            this.style.border = `1px solid ${this.badgeBackgroundColor}`;
+            this.style.border = `1px solid ${
+                this.borderColor ? this.borderColor : this.badgeBackgroundColor
+            }`;
```

Comment:
also would it be possible to move this in a getter?

Reply:
Moved to getter `computedBorderStyle()`